test/common/canonical-iam-policy.ts

```diff
@@ -0,0 +1,228 @@
+import { normalizePolicy } from '../../src/services/canonical-iam-policy';
```

Comment:
common tests should be running and they are not, we should fix the CI job and make sure those test pass before landing